admin_guide/out_of_resource_handling.adoc

```diff
@@ -116,21 +119,20 @@ To configure hard eviction thresholds, the following flag is supported:
 The node evaluates eviction thresholds per its housekeeping interval. It
 monitors every 10 seconds and the value can not be modified.
 
-* `housekeeping-interval` is the interval between container housekeepings.
+* `housekeeping-interval`: the interval between container housekeepings.
```

Comment:
just drop this since its not an actual flag that can be modified at this time.  sorry for the error on my original doc.